sklearn/ensemble/gradient_boosting.py

```diff
@@ -1108,6 +1143,24 @@ def _fit_stages(self, X, y, y_pred, sample_weight, random_state,
                 early_stopping = monitor(i, self, locals())
                 if early_stopping:
                     break
+
+            # We also provide an early stopping based on the score from
+            # validation set (X_val, y_val), if n_iter_no_change is set
+            if self.n_iter_no_change is None:
+                continue
```

Comment:
continue in the middle of a block is often hard to find. I wonder whether the code would be slightly more readable if you made this `is not None` and indented the rest. Definitely a cosmetic nitpick, though.